### PR TITLE
Questionaries

### PR DESCRIPTION
**Pull request by bart-merix**

### CODE REVIEW

File: questionnaires/admin.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+class SurveyAdmin(ModelAdmin):
+    model = Survey
+    menu_label = "Surveys"
+    menu_icon = "doc-full"
+    add_to_settings_menu = False
+    exclude_from_explorer = False
+
+
+modeladmin_register(SurveyAdmin)
```
Comment (bart-merix) [inline questionnaires/admin.py:31]: Do we want to group surveys and polls as 1 big family on wagtail admin?

File: questionnaires/migrations/0003_auto_20210615_0552.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Generated by Django 3.1.12 on 2021-06-15 05:52
```
Comment (mwrobel-merixstudio) [inline questionnaires/migrations/0003_auto_20210615_0552.py]: there are 3 migration files. its good to make only one migration per module per PR.

you can revert migrations, delete these files and re-create migration - so there will be only one migration file

File: questionnaires/models.py [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+        on_delete=models.PROTECT,
+        help_text=_("Optional page to which user after questions can go"),
+    )
+    allow_anonymous_submissions = models.BooleanField(
+        default=True,
+        help_text=_(
+            "Check this to allow users who are NOT logged in to complete surveys."
+        ),
+    )
+    # TODO create validation if set to false
```
Comment (mwrobel-merixstudio) [inline questionnaires/models.py]: what about this "todo"? should it be done now?

File: questionnaires/views.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        "poll/results.html",
+        context,
+    )
+
+
+class VoteFormView(FormView):
+    template_name = "poll/poll.html"
+    form_class = VoteForm
+
+    # TODO: vote after login doesn't work
```
Comment (mwrobel-merixstudio) [inline questionnaires/views.py]: do we know why? is there subtask created to remember to handle it later?

**Review by mwrobel-merixstudio — APPROVED** [review]
looks fine to me - there is some "todos" in code - so if they cannot be handle in this PR - please create subtasks so we know there is still some work to do

**Pull request closed without merging**